input/sources/tev-000095.yaml

```diff
@@ -1,5 +1,17 @@
 source_id: 95
 
+common_name: CTB 37B
```

Comment:
I think we messed up the id for RXJ1713.7-3946. It should be id_96 not id_95.
Id_95 is identical to CTB 37B, see here
http://tevcat2.uchicago.edu/sources/QYavG5
and here
http://tevcat.uchicago.edu/?mode=1&showsrc=110
id_96 has no CTB-like code so I erased that in sources/tev-000096.yaml